llmlite/llms/model.py

```diff
@@ -73,20 +73,24 @@ def load_with_vllm(
         }
         return cls(model_name_or_path, **config_args)
 
-    def completion(self, messages: List[ChatMessage], **kwargs) -> Optional[str]:
+    def completion(self, messages: List[List[ChatMessage]], **kwargs) -> Optional[Union[str, List[str]]]:
         if self.backend_runtime is None:
             raise Exception("Please implement the completion() additionally.")
-
-        prompt = self.prompt(self.model_name_or_path, messages, **kwargs)
-        if prompt is None:
+        
+        prompt = []
```

Comment:
I think we can be simple here, the general idea is if we use vllm as backend, then we always assume the inputs are batch messages, and the output should always be an array, I think this is consistent with vllm. Then as an user, when using vllm backend, he will always get a list of outputs.

Reply:
Maybe we can use this strategy